src/main/scala/viper/gobra/frontend/info/implementation/typing/ghost/GhostMemberTyping.scala

```diff
@@ -33,8 +34,11 @@ trait GhostMemberTyping extends BaseTyping { this: TypeInfoImpl =>
     if (member.spec.isPure) {
       isSingleResultArg(member) ++
         isSinglePureReturnExpr(member) ++
-        isPurePostcondition(member.spec)
-    } else noMessages
+        isPurePostcondition(member.spec) ++
+        nonVariadicArguments(member.args)
+    } else {
+      wellDefVariadicArgs(member.args)
```

Comment:
I would check this in MemberTyping. Also, maybe the isPure function has to be updated for calls.

Reply:
| Also, maybe the isPure function has to be updated for calls.

Why? It seems to me that the `isPure` results should be the same, meaning that calls to non-pure methods are not pure and calls to pure methods cannot have variadic arguments. My point here is that the presence of variadic parameters does not affect the purity of a method call. Am I missing anything?